docs/hello_nextflow/02_hello_world.md

```diff
@@ -822,13 +833,13 @@ In most cases, when we run on multiple inputs, the input values are contained in
 
 Nextflow offers a variety of predefined operators and functions for reading data in from common file formats and applying text transformations. In this example, we used the [`fromPath()`](https://www.nextflow.io/docs/latest/channel.html#frompath) channel factory with the [`splitText()`](https://www.nextflow.io/docs/latest/operator.html#splittext) operator to read each line as a separate value, then we used a [closure](https://www.nextflow.io/docs/latest/script.html) to apply the [`trim()`](https://www.tutorialspoint.com/java/java_string_trim.htm) method to strip the newline (`\n`) character from each element.
```

Comment:
This is too much infodump up front-- we want to parcel out this information at each step.